rinex-cli/src/positioning/interp/time.rs

```diff
@@ -0,0 +1,220 @@
+use crate::cli::Context;
+use gnss_rtk::prelude::{Epoch, SV};
+use std::collections::HashMap;
+
+use super::Buffer as BufferTrait;
+
+struct Buffer {
+    inner: Vec<(Epoch, f64)>, //TODO
+}
+
+impl BufferTrait<f64> for Buffer {
```

Comment:
I don't think the trait actually makes this code clearer, a wrapper over `Vec<f64>` (or `[f64; 3]' or `StaticVec<f64; 3>`) might be sufficient.

Reply:
I was interested in having this Trait because we need both spatial and time interpolation in this topic, therefore they share the same logic. Especially how we manage in the internal fifo against data gaps.

Note that data gaps are not correctly handled in the current implementation. This proposed form is a first solution, probably too stringent (does not tolerate any gaps, since we reset the fifo when a gap appears), but we will improve that in the future (simple detail)

Reply:
I'm missing the larger context, but a `struct Buffer(Vec<f64>);` would work just as well and be easier to follow. The trait has a single implementation right now, and it seems unlikely it will get others, but I might be wrong.